scalapact-http4s-0-20/src/main/scala/com/itv/scalapact/http4s20/impl/Http4sRequestResponseFactory.scala

```diff
@@ -111,7 +111,7 @@ object Http4sRequestResponseFactory {
 
 object HeaderImplicitConversions {
   implicit def mapToHeaderList(headerMap: Map[String, String]): Headers =
-    Headers.apply(headerMap.toList.map(t => Header(t._1, t._2)))
+    Headers(headerMap.toList.map(t => Header(t._1, t._2)))
```

Comment:
I added that after the last review to fix a weird bug where the compiler wasn't selecting the right apply method. If the testing process fails I might have to put it back.

Reply:
I did a quick manual test to see if it doesn't crash and run the tests too, seemed to be fine

Reply:
Thanks, it was a weird one that only appeared near the end of the full test process. Running the script now.

Reply:
No errors, perhaps they fixed something in the final release.